sdk/monitor/query/azlogs/constants.go

```diff
@@ -54,8 +54,8 @@ const (
 	versionsV1 versions = "v1"
 )
 
-// possibleVersionsValues returns the possible values for the versions const type.
-func possibleVersionsValues() []versions {
+// possibleversionsValues returns the possible values for the versions const type.
+func possibleversionsValues() []versions {
```

Comment:
Just checking, you lost your camel casing for 'versions'

Reply:
This is correct. There was a bug in generator before where this private const was generating as private, except for this func. I had to add a custom transform to the transforms.go file, so that caused the old casing.

Joel was awesome and fixed the bug, and now the generator is generating this correctly.